less/navigationBar.less

```diff
@@ -745,6 +743,12 @@
       min-height: @urlbarFormHeight;
       min-width: @urlbarFormHeight;
       -webkit-app-region: no-drag;
```

Comment:
@cezaraugusto I thought this is required for `.addPublisherButtonContainer` too, to make it possible to control remotely, right?